v22.2/fips-compliance.md

```diff
@@ -0,0 +1,162 @@
+
+When a cryptographic module or library has a FIPS 140-2 certificate, it has been tested and formally validated by the U.S. and Canadian Governments.
+
+The standard CockroachDB binaries rely on the Go language for cryptographic functions and do not use OpenSSL libraries installed in the operating system. Go's cryptographic libraries are not FIPS-validated.
+
+FIPS-compliant CockroachDB binaries are built using [Red Hat's Go FIPS with OpenSSL toolchain](https://github.com/golang-fips/go), which contains the necessary modifications for the Go crypto library to use an external cryptographic library in a FIPS compliant way. FIPS-compliant CockroachDB runtimes use the host operating system's OpenSSL libraries and commands, rather than Go's cryptographic libraries. When the installed OpenSSL has a FIPS 140-2 certificate and [FIPS mode is enabled in the Linux kernel](#enable-fips-mode-in-the-linux-kernel), the CockroachDB runtime is compliant with FIPS 140-2. FIPS-compliant binaries and Docker images are available for CockroachDB v22.2.7 and above. CockroachDB FIPS-compliant runtimes run only on Intel 64-bit Linux systems.
+
+### FIPS-compliant features
+
+When you use a FIPS-compliant CockroachDB runtime and each cluster node's OpenSSL is FIPS-validated and configured correctly, the following features have been tested to ensure that their cryptographic operations comply with FIPS 140-2:
+
+- [Encryption At Rest](security-reference/encryption.html#encryption-keys-used-by-cockroachdb-self-hosted-clusters)
+- [Encrypted Backups](take-and-restore-encrypted-backups.html)
+- [Change Data Capture to Kafka over TLS](create-changefeed.html#query-parameters)
+- [Certificate-based Node-to-Node and Client-to-Node Authentication](authentication.html)
+- [SASL SCRAM-SHA-256 Password Authentication](security-reference/scram-authentication.html)
+- [Built-in SQL cryptographic functions](functions-and-operators.html#cryptographic-functions)
+
+  {{site.data.alerts.callout_danger}}
+  In a FIPS-compliant system, Cockroach Labs recommends that you avoid using cryptographic operations that are not supported by FIPS 140-2. For example, generating an MD5 hash is not FIPS-compliant, because MD5 is not a FIPS-compliant algorithm. Use algorithms and functions that do not comply with FIPS at your own risk.
+  {{site.data.alerts.end}}
+
+This page shows how to configure {{ site.data.products.core }} to comply with FIPS 140-2 using Red Hat's FIPS-validated OpenSSL package. The FIPS-validated Docker image for CockroachDB comes with Red Hat's OpenSSL libraries pre-installed and configured.
```

Comment:
Let's have a separate section that talks about which artifacts can be downloaded, what they contain and where to find them. This paragraph would go there.